lib/usecases/run/StartRunUseCase.js

```diff
@@ -91,7 +91,8 @@ class StartRunUseCase {
             runType = newType.result;
         }
         run.runTypeId = runType.id;
-        return { run: await RunRepository.insert(runAdapter.toDatabase(run)) };
+        return { run: await runService.create(run) };
+        // Return { run: await RunRepository.insert(RunAdapter.toDatabase(run)) };
```

Comment:
Please remove this commented out code. Besides that, nice refactoring 👍🏻

Reply:
done